### PR TITLE
spelling and labels updates  policycompass/policycompass#453

### PR DESCRIPTION
**Pull request by mmilaprat**
Update labels in modal windows (link datasets and link events)

Related with issue policycompass/policycompass/issues/453

### CODE REVIEW

File: app/modules/visualization/controllers/visualizationControllers.js [hunk 1]
```diff
@@ -4288,8 +4297,12 @@ angular.module('pcApp.visualization.controllers.visualization', [
             
             var datasetsURL = $routeParams.datasets;
 			
+
 			
+			$scope.setDisableLinkDataset(false);
             if (datasetsURL) {
+            	$scope.disableLinkDataset = true;
```
Comment (cbotsikas) [inline app/modules/visualization/controllers/visualizationControllers.js]: @mmilaprat, this is unnecessary due to the line below

**Comment by cbotsikas** [comment]
I think your [3rd commit](https://github.com/mmilaprat/policycompass-frontend/commit/49aed1bb910c9e60861e150eedb86211a8c4e7f8) is not necessary based on my findings (https://github.com/policycompass/policycompass/issues/430#issuecomment-193503967) and also contains a duplicate line (see my inline comment https://github.com/policycompass/policycompass-frontend/pull/315#discussion_r55248091). The 1st one is fine and the 2nd works, but to be honest, i've no idea why :).

I'll close this PR. 
Create a new PR including the [1st commit](https://github.com/mmilaprat/policycompass-frontend/commit/c0082b4cbfb73cc807f7f030f174972ab414a9c2). I or @FabiApfelkern will merge it as soon as you do the PR. Then you can focus on issue https://github.com/policycompass/policycompass/issues/430 where you can include your [2nd commit](https://github.com/mmilaprat/policycompass-frontend/commit/61862796d10ee4c6421d7212294b66260034038d) (actually you should remove the line instead of commenting it out)

**Comment by mmilaprat** [comment]
Can you accept this PR and then I try to solve the initial issue according your suggestions?

**Comment by mmilaprat** [comment]
forget it. I'll do another PR

**Pull request closed without merging**